flask_peewee/rest/__init__.py

```diff
@@ -605,6 +613,23 @@ def api_exportable(self):
             } for h, c, _ in self.export_columns]
         })
 
+    def api_detail_json(self, pk, field, path):
+        path = path.split('/')
+
+        if field not in self.editable_json_fields:
```

Comment:
should there be a restriction that you can only modify json fields using the json endpoints or should you be able to modify them in a normal edit as well?